src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -106,9 +129,17 @@ object Scope {
   }
 
   @tailrec
-  def fullyQualifiedNameBuilder(scope: Option[Scope], fqn: String, fullDisplay: Boolean): String = scope match {
-    case None => fqn.tail //Strip away the first "." of the name
-    case Some(x: Scope) => fullyQualifiedNameBuilder(x.parent, (if (fullDisplay || x.appearsInFQN) s".${x.name}" else "") + fqn, fullDisplay)
+  def fullyQualifiedNameBuilder(scope: Option[Scope], fqn: String, fullDisplay: Boolean, leaf: Boolean): String = {
+    scope match {
+      case None =>
+        fqn.tail //Strip away the first "." of the name
```

Comment:
No worries, will do. And "in the wild" I've mostly seen that order too.

However, in my last reactive scala class, especially with recursive functions like this one, a lot of the example code listed the breakout first, then the more complex iteration below. So `None`, `Nil`, etc. was usually the first `match` entry.